arch/riscv/custom/thead/cache_xtheadcmo.c

```diff
@@ -83,7 +84,8 @@ int arch_icache_invd_range(void *addr_in, size_t size)
 		"fence\n"
 		"fence.i\n"
 	);
-	for (uintptr_t i = addr; i < addr + size; i += CONFIG_ICACHE_LINE_SIZE) {
+	for (uintptr_t i = addr; i < ROUND_UP(addr + size, CONFIG_DCACHE_LINE_SIZE);
```

Comment:
it seems like it's a version before the `D` -> `I` fixup

Reply:
yes